src/pdn/doc/define_pdn_grid.md

```diff
@@ -0,0 +1,104 @@
+## define_pdn_grid
+
+### Synopsis
+For specifying a power grid over the stdcell area
+```
+  % define_pdn_grid
+    -type stdcell \
```

Comment:
might be better to change `-type` to a flag like: `-macro` since it is basically binary and assume its stdcell when not specified.

Reply:
Added -macro flag, if there is no -macro flag then the grid applies to stdcell area.